tms_objects2.erb

```diff
@@ -274,6 +274,14 @@ xsi:schemaLocation="http://www.lido-schema.org http://www.lido-schema.org/schema
         <% end %>  
         <% end %>   <%# ConAltNames |a| %>
                   </lido:nameActorSet>
```

Comment:
I take back what I said.  There should be logic like in coboat:
{if $DC->Nationality != ''} 
for nationality actor. Find an object where there isn't a nationality and see that nationalityActor empty nationality is suppressed.

Reply:
Sorry, there is more here. DC.Nationality isnt a good indicator as this field is being used for more that nationality. Instead use like <% if ConAttributes.length > 0 %>